tuf/keys.py

```diff
@@ -74,11 +75,11 @@
 # if 'pynacl' is unavailable.
 _SUPPORTED_ED25519_CRYPTO_LIBRARIES = ['ed25519', 'pynacl']
 
-# 'pycrypto' is the only currently supported library for general-purpose
-# cryptography, with plans to support pyca/cryptography.
+# 'pycrypto' and 'cryptography' are the only currently supported library for
+# general-purpose cryptography, with plans to support pyca/cryptography.
```

Comment:
Resolved by [commit](https://github.com/vladimir-v-diaz/tuf/commit/f6d7302293d23ab2094f865464ebbf17b2633168) in [PR #311](https://github.com/theupdateframework/tuf/pull/311)